package.json

```diff
@@ -111,7 +111,7 @@
     "symbol-hd-wallets": "0.14.1-alpha-202103051108",
     "symbol-post-launch-optin-module": "1.0.1",
     "symbol-qr-library": "0.14.1-alpha-202103081047",
-    "symbol-sdk": "2.0.2",
+    "symbol-sdk": "/Users/bahadir/dev/code/repo/clients/nem/symbol/symbol-sdk-typescript-javascript-priv/symbol-sdk-2.0.3.tgz",
```

Comment:
@yilmazbahadir Need to fix this. :)

Reply:
that must already have been removed (upgrade to 2.0.3)